ncc.js

```diff
@@ -26,6 +26,7 @@ async function build() {
 
 async function compile(entry) {
   const { code, map, assets } = await ncc(entry, {
+    externals: ['@expo/fingerprint', 'module', 'sqlite3'],
```

Comment:
Let's move this only to apply when building these two actions. Right now, I think it also gets applied to the other actions, which may be unexpected.

One thing we can do is to create an externals dictionary, with the `entry` as key.